styles/components/input.css

```diff
@@ -0,0 +1,14 @@
+:root{
```

Comment:
Move all color variables in one single CSS file, instead of defining them in different files